cli/azd/pkg/input/console.go

```diff
@@ -133,6 +136,26 @@ func (c *AskerConsole) Message(ctx context.Context, message string) {
 	}
 }
 
+func (c *AskerConsole) WarnForFeature(ctx context.Context, key alpha.FeatureId) {
```

Comment:
I actually like this quite a lot! We end up avoiding needing additional components for outputting warning messages